src/psyclone/dynamo0p3.py

```diff
@@ -3262,66 +4320,26 @@ def gen_code(self, parent):
                                      rhs=omp_function_name+"()"))
 
         # Initialise any stencil maps
-        self.stencil.initialise_stencil_maps(invoke_sub)
+        self.stencil.initialise(invoke_sub)
+
+        self.orientation.initialise(invoke_sub)
```

Comment:
This looks lonely without a comment.

Reply:
I've gone the whole hog and whacked all of these inside a loop too.